File: Leetcode/PythonAnswers/Easy/1437KLengthsapart.py
class Solution:
    def kLengthApart(self, nums: List[int], k: int) -> bool:
        if k == 0:
            return(True)
        for i in range(len(nums)):
            if nums[i] == 1:
                for j in range(len(nums)):
                    if nums[j] == 1 and j > i and ((j-i)<=k):
                        return False
        return(True)
                        
#Accepted answer

File: Leetcode/Java/Easy/1437KLengthsapart.java
class Solution {
    public boolean kLengthApart(int[] nums, int k) {
        if (k == 0){
            return(true);
        }
        for (int i = 1; i < nums.length; i++) {

            if (nums[i] == 1) {
                for (int j = 1; j < nums.length; j++){
                    if (nums[j]== 1 && (j > i) && ((j - i -1) < k)){
                        return(false);
                    }
                }
            } 

        }
        return(true);
}
}
//Not done yet